src/app/topbar-reminders/topbar-reminders.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { TopbarService } from '../shared/services/topbar.service';
+import { WalletService } from '../shared/services/wallet/wallet.service';
+import { throwIfEmpty } from 'rxjs/operators';
+import { WalletDetails } from '../shared/services/wallet/wallet-cooperative/wallet-cooperative-wallet.service';
+import { timingSafeEqual } from 'crypto';
+
+@Component({
+  selector: 'app-topbar-reminders',
+  templateUrl: './topbar-reminders.component.html',
+  styleUrls: ['./topbar-reminders.component.css']
+})
+export class TopbarRemindersComponent implements OnInit, OnDestroy {
```

Comment:
Rename it to `UserStateReminderComponent`. Well, it's a bad practice to name things where they are located. It's better to indicate what this component does. In this case, it reminds the user of its current state.